addons/repair/functions/fnc_addRepairActions.sqf

```diff
@@ -36,7 +36,7 @@ if (_type in _initializedClasses) exitWith {};
 private _hitPointsAddedNames = [];
 private _hitPointsAddedStrings = [];
 private _hitPointsAddedAmount = [];
-private _processedHitpoints = [];
+private _processedSelections = [];
```

Comment:
I also changed hitpoints to selections here because hitpoints are always unique (I hope) and there is no sense to check them. Selections as we see can be the same for different hitpoints and should be checked.